allan_variance.py

```diff
@@ -7,25 +7,25 @@
 
 
 def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
-                   n_clusters=100):
-    """Compute Allan variance (AV).
-    
+                   n_clusters=100, input_mode="increment"):
```

Comment:
I think `input_type` is a better name. 

I'm quite neutral whether to include this parameter at all, but don't mind.

It should be documented too.